aws/resource_aws_sfn_state_machine.go

```diff
@@ -53,20 +84,32 @@ func resourceAwsSfnStateMachine() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
 			"tags": tagsSchema(),
+
+			"type": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
Do we need to have `ForceNew: true,` added here?
Once a state machine is created the type cannot change.  When attempting to change the type the resource should be destroyed and rebuilt.
https://docs.aws.amazon.com/step-functions/latest/dg/concepts-standard-vs-express.html